styles/styles.css

```diff
@@ -0,0 +1,115 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans' rel="stylesheet");
+
+.container {
+  font-family: 'Noto Sans', sans-serif;
+  text-align: center;
+  width: 100%;
+  }
+
+ul {
+  list-style-type: none;
+}
+
+nav ul {
+  text-align: left;
+  font-weight: bold;
+  display: grid;
+  grid-template-columns: 60% 20% 20%;
+}
+
+.top {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.top a {
+  margin: 3em;
+  border: solid #9ea1a5 thin;
+  border-radius: 10px;
+  text-decoration: none;
+  color: black;
+  padding: 10px;
+  font-size: 1em;
+}
+
+.quote {
```

Comment:
I really like how your quotes are laid out.  Very easy to read and flows well in its container.  One question:  What do you think about using the pseudo-elements ::before and ::after to add quotes and a dash (like in the wireframe)?